random picture/app.js

```diff
@@ -0,0 +1,6 @@
+var imagesArray = ["https://source.unsplash.com/1600x900", "https://source.unsplash.com/1600x900", "https://source.unsplash.com/1600x900", "https://source.unsplash.com/1600x900", "https://source.unsplash.com/1600x900"];
+
+function newRandomImage() {
+    let num = Math.floor(Math.random() * 5);
+    document.canvas.src = imagesArray[num];
+}
```

Comment:
Nice work, but you should create the image tog from the JS, not in the index.html